Test/Mono.Cecil.Tests/BaseTestFixture.cs

```diff
@@ -119,19 +119,21 @@ abstract class TestCase {
 		public readonly bool ReadOnly;
 		public readonly Type SymbolReaderProvider;
 		public readonly Type SymbolWriterProvider;
+		public readonly BaseAssemblyResolver AssemblyResolver;
```

Comment:
That's alright, if the user supplies its own IAssemblyResolver, we can expect that it will be fully configured. The AddSearchDirectory should only be called if a DefaultAssemblyResolver is created.